py_cui/__init__.py

```diff
@@ -436,12 +437,12 @@ def set_widget_border_characters(self, upper_left_corner, upper_right_corner, lo
         self._logger.debug(f'Set border_characters to {self._border_characters}')
 
 
-    def get_widgets(self):
+    def get_widgets(self) -> Dict[int,Optional['py_cui.widgets.Widget']]:  
         """Function that gets current set of widgets
 
         Returns
         -------
-        widgets : dict of str -> widget
+        widgets : dict of int -> widget 
```

Comment:
Thanks for switching the type here, looks like I missed it

Reply:
I might not have changed this in all places, though.